bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -1931,4 +1932,17 @@ private function addGotenbergNode(ArrayNodeDefinition $rootNode): void
             ->end()
         ->end();
     }
+    private function addChromiumNode(ArrayNodeDefinition $rootNode): void
+    {
+        $rootNode
+            ->children()
+            ->arrayNode('chromium')
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('base_url')
+                ->defaultValue('chromium:3000')
```

Comment:
```suggestion
                ->scalarNode('uri')
                ->defaultNull()
```
I'd leave the default null here.
If null, we're using the build-in auto-detect: 
https://github.com/chrome-php/chrome/blob/1.8/src/BrowserFactory.php#L108

If configured, we connect directly to the configured `$uri`.